data/ai/micro_ais/scenarios/messenger_escort.cfg

```diff
@@ -118,7 +118,7 @@
 
         # waypoints for AI
         {PLACE_IMAGE "scenery/signpost.png" 31 20}
-        {SET_LABEL 31 20 _"Messanger Waypoint 1"}
+        {SET_LABEL 31 20 _"Messenger Waypoint 1"}
```

Comment:
Another translatable that would need to be pofixed. Though, given the timing, I dunno, maybe not...